zingolib/src/wallet/transaction_record.rs

```diff
@@ -445,10 +445,10 @@ impl TransactionRecord {
             0
         };
 
-        // Outgoing metadata was only added in version 2
-        let outgoing_metadata =
-            zcash_encoding::Vector::read(&mut reader, |r| OutgoingTxData::read(r))?;
-
+        let outgoing_metadata = match version {
```

Comment:
does this mean that version is bumping to 24?

Reply:
Did I not update the version number?? How the hell did that pass the test suite??? Fixing that now, oh my god